Tests/Form/Widget/FormSonataFilterChoiceWidgetTest.php

```diff
@@ -64,7 +63,7 @@ public function testDefaultValueRendering()
 
     protected function getParentClass()
     {
-        if (version_compare(Kernel::VERSION, '2.8.0', '>=')) {
+        if (class_exists('Symfony\Component\Form\Extension\Core\Type\RangeType')) {
```

Comment:
Maybe we could have a feature detection class for that, with methods like `formComponentUsesChoicesAsValues()` that would abstract away these code snippets…